### PR TITLE
Bump the version to 2.1

### PR DESCRIPTION
**Pull request by ankitkala**
Signed-off-by: Ankit Kala <ankikala@amazon.com>

### Description
Bump the version to 2.1

### Issues Resolved
[430](https://github.com/opensearch-project/cross-cluster-replication/issues/430)
 
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by saikaranam-amazon** [comment]
common-utils are not yet built for 2.1
```
* What went wrong:
Could not determine the dependencies of task ':thirdPartyAudit'.
> Could not resolve all files for configuration ':runtimeClasspath'.
   > Could not resolve org.opensearch:common-utils:2.1.0.0-SNAPSHOT.
     Required by:
         project :
      > Could not resolve org.opensearch:common-utils:2.1.0.0-SNAPSHOT.
         > Could not get resource 'https://plugins.gradle.org/m2/org/opensearch/common-utils/2.1.0.0-SNAPSHOT/common-utils-2.1.0.0-SNAPSHOT.pom'.
            > Could not GET 'https://repo.gradle.org/artifactory/jcenter/org/opensearch/common-utils/2.1.0.0-SNAPSHOT/common-utils-2.1.0.0-SNAPSHOT.pom'. Received status code 409 from server: Conflict

```

**Comment by saikaranam-amazon** [comment]
common-utils package for reference: https://github.com/opensearch-project/common-utils/blob/main/build.gradle#L9

File: build.gradle [hunk 1]
```diff
@@ -140,7 +140,7 @@ repositories {
     mavenCentral()
     maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
     maven { url "https://plugins.gradle.org/m2/" }
-    maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+    maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }
```
Comment (saikaranam-amazon) [inline build.gradle:143]: Are we publishing `SNAPSHOTS` artifacts to this one?

**Review by saikaranam-amazon — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (ankitkala) [reply build.gradle:143]: Nope. This is for fetching Lucene related artifacts.

**Review by gbbafna — APPROVED** [review]

**Pull request merged**